cocos/serialization/ccon.ts

```diff
@@ -230,6 +229,6 @@ export class BufferBuilder {
     }
 }
 
-legacyCC.internal.parseCCONJson = parseCCONJson;
-legacyCC.internal.decodeCCONBinary = decodeCCONBinary;
-legacyCC.internal.CCON = CCON;
+cclegacy.internal.parseCCONJson = parseCCONJson;
+cclegacy.internal.decodeCCONBinary = decodeCCONBinary;
```

Comment:
Any reason of renaming `legacyCC` to `cclegacy`?

Reply:
@PPpro said, it is reported as `cclegacy` outside.